src/thumbnailTrack.ts

```diff
@@ -0,0 +1,31 @@
+import { MakeRequired } from './utils';
+
+/**
+ * Represents a custom subtitle track source that can be added to `SourceConfig.subtitleTracks`.
+ */
+export interface ThumbnailTrack {
+  /**
+   * The URL to the timed file, e.g. WebVTT file.
+   */
+  url?: string;
+  /**
+   * The label for this track.
+   */
+  label?: string;
+  /**
+   * The unique identifier for this track. If no value is provided, a random UUIDv4 will be generated for it.
+   */
+  identifier?: string;
```

Comment:
Even `identifier` could be removed because there can only be one thumbnail track.
Feel free to use a constant value for this.

Reply:
Done